website/docs/api/large-language-models.mdx

```diff
@@ -32,13 +24,26 @@ as well as through task-specific component factories: `llm_ner`, `llm_spancat`,
 > llm = nlp.add_pipe("llm", config=config)
 >
 > # Construction via add_pipe with a task-specific factory and default GPT3.5 model
-> llm = nlp.add_pipe("llm-ner")
+> llm = nlp.add_pipe("llm_ner")
+>
+> # Construction via add_pipe with a task-specific factory and custom model
+> llm = nlp.add_pipe("llm_ner", config={"model": {"@llm_models": "spacy.Dolly.v1", "name": "dolly-v2-12b"}})
```

Comment:
Wouldn't this fail due to a component with the same name twice?

Reply:
So these snippets are supposed to be independent, like they show different ways of initializing the same `llm` component (it's clearer if you look at how this is rendered)